### DISCUSSION TITLE
What do J/K do? Everything disappears. Where can I find and define the keys on PC?

### DISCUSSION THREAD

**Discussion by elandorr** [koreader/koreader#11879] [Q&A]
Pressing J/K seems to toggle something. The screen goes blank, but comes back when you press it a couple of times.
Tried a PDF.
I'd like to customize keys in general to make it less touch-y and more PC-friendly.

**Comment by Frenzie** [comment]
It looks like J and K are rotate left and right, but it doesn't seem to be working at a glance.
